index.qmd

```diff
@@ -17,6 +17,14 @@ With this guide, we provide the landscape of cloud-optimized geospatial formats
 
 Just putting data on the cloud does not solve the big geospatial data problem. Massive archives of data must be available via subsetting services in order for users to work with the data in-memory. Traditional geospatial formats are optimized for on-disk access via small internal chunks. The introduction of a network introduces  latency and the number of requests must considered. The file format must support subsetted access via adressable chunks, internal tiling or both. These characteristics allow for parallelized and partial reading.
 
+## Data Type to Traditional to Cloud-Optimized Geospatial File Format Table
+
+The diagram below depicts how some of the cloud-optimized formats discussed in this guide are cloud-optimized formats of traditional geospatial file formats.
+
+Not depicted is how GeoTIFF, FlatGeobuf, and GeoParquet are geospatial versions of the non-geospatial file formats TIFF, binary and parquet, respectively.
```

Comment:
binary or [flatbuffers](https://github.com/google/flatbuffers) (no one really knows what those are, etc [protobuf](https://protobuf.dev/))

Reply:
Are you suggesting to reword this somehow? I was thinking not to go into the nitty gritty of flatgeobuf encoding here.

Reply:
yes, binary is too vague - technically all the formats are binary (ie not plaintext)

Reply:
I changed it to flatbuffers and totally get your point about it binary being too vague, I was trying to avoid being too in the weeds for overview content.

Reply:
I agree that "binary" is too vague, but I'm not a huge fan of referencing Flatbuffers in the highest overview page either... TIFF and Parquet are both formats that _end users_ would use directly, whereas Flatbuffers is a format that only developers would use in order to build their own data formats. I.e. there's no way to read a "flatbuffer file"; like with protobuf you'd need to create a schema for your specific data format, generate code for each binding language to read/write it, and then use that specific API with your data buffers.